formfactory/templates/formfactory/inclusion_tags/form_detail.html

```diff
@@ -1,4 +1,4 @@
-<form class="Form" method="POST">{% csrf_token %}
+<form class="Form" method="POST" action="{{ object.post_to|default:"" }}">{% csrf_token %}
```

Comment:
Not needed anymore. Merge with develop please. `action="."` is all that is required.